CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ $ pnpm install
 $ pnpm start
 ```
 
-> Please use [PNPM][pnpm-url] while working on this project.
+> Please use [PNPM][pnpm-url] version specified in the `package.json` file while working on this project. You can use [corepack][corepack-url] to manage multiple pnpm versions.
```

Comment:
We can drop the version mention from here. When a new contributor runs `npm install`, it will check the `engines` automatically, throwing if `pnpm` is not installed or the non-matching version is installed. I do hope that's the behavior of `engines`. 

Also, I'd drop any recommendations of third-party tooling to help manage package managers. Those are implicit knowledge and are not specific to MSW.

Reply:
Instead, to ease the contributing experience, we can:

- Clearly state that "Please use PNPM version 7.12".
- Include a curl URL that installs the required version directly. Following the installation from PNPM's docs will always lead to the latest version installed, which will get people confused.

Reply:
I have now updated that file to point to the official pnpm docs for a guide on how to install a specific version and added that `7.12` is the one to use, removed the `corepack` mention

Reply:
I had a brief look at the codesandbox CI docs but didn't find anything of how we could pin the `pnpm` version there.

Some issues on their repo mention the same problem and a possible workaround is to install a specific version of `pnpm` through a custom `installCommand` script, what do you think about that?